src/System.Private.CoreLib/shared/System/Range.cs

```diff
@@ -64,9 +105,13 @@ public override string ToString()
             return new string(span.Slice(0, pos));
         }
 
-        public static Range Create(Index start, Index end) => new Range(start, end);
-        public static Range FromStart(Index start) => new Range(start, new Index(0, fromEnd: true));
-        public static Range ToEnd(Index end) => new Range(new Index(0, fromEnd: false), end);
-        public static Range All() => new Range(new Index(0, fromEnd: false), new Index(0, fromEnd: true));
+        /// <summary>Create a Range object starting from start index to the end of the collection.</summary>
+        public static Range StartAt(Index start) => new Range(start, new Index(0, fromEnd: true));
+
+        /// <summary>Create a Range object starting from first element in the collection to the end Index.</summary>
+        public static Range EndAt(Index end) => new Range(new Index(0), end);
+
+        /// <summary>Create a Range object starting from first element to the end.</summary>
+        public static Range All { get { return new Range(new Index(0), new Index(0, fromEnd: true)); }}
```

Comment:
Consistent style with StartAt/EndAt:

```suggestion
        public static Range All => new Range(new Index(0), new Index(0, fromEnd: true));
```

Reply:
will do that

Reply:
Also use `Index.Start`/`End` properties.